Documentation/roadmap.md

```diff
@@ -0,0 +1,30 @@
+Our goal is to create a set of world-class APIs and tooling that will support a rich .NET IoT ecosystem.  Below shows a structure of components as part of our roadmap.
+
+**NOTE**: Not all components listed below are included with current previews.
+
+![](images/DotNetIotRoadmapComponents.png)
+
+We have the following deliverables on our intermediate-term roadmap (not in order of completion):
```

Comment:
I'd remove the brackets or reword to something like `not in any particular order`